src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,29 @@
+package calculator;
+
+import java.util.Scanner;
+
+public class StringCalculator {
```

Comment:
```suggestion
public class StringCalculatorApplication {
```
main이 있는 클래스명은 application을 붙여주면 다른 역할을 담당하는 클래스와 구분이 더 명확하게 될거에요
StringCalculator만 쓰면 도메인 객체로 보입니다